6.x/crud-fields.md

```diff
@@ -1401,6 +1401,87 @@ Input preview:
 
 <hr>
 
+<a name="dropzone"></a>
+### dropzone <span class="badge badge-pill badge-info">PRO</span>
+
+Show a [Dropzone JS Input](https://docs.dropzone.dev/).
+
+**Step 0.** Make sure the db column can hold the amount of text this field will have. For example, for MySQL, `VARCHAR(255)` might not be enough all the time (for 3+ files), so it's better to go with `TEXT`. Make sure you're using a big column type in your migration or db.
+
+**Step 1:** Add the `DropzoneOperation` to your `CrudController`
+
+```php
+class UserCrudController extends CrudController
+{
+    use \Backpack\CRUD\app\Http\Controllers\Operations\CreateOperation;
+    // ... other operations
+    use Backpack\Pro\Http\Controllers\Operations\DropzoneOperation;
+}
+```
+
+**Step 2:** Add the field in CrudController
+
+```php
+$this->crud->addField([   
+    'name'  => 'photos',
+    'label' => 'Photos',
+    'type'  => 'dropzone',
+
+    // optional configuration.
+    // check available options in https://docs.dropzone.dev/configuration/basics/configuration-options
+    'configuration' => [
+        'parallelUploads' => 2,
+    ]
+]);
+```
+
+**Step 3:** Configure the file upload process.
+
+At this point you have the dropzone field showing up, and the ajax routes setup to upload/delete files, but the process is not complete. 
+Your files are now only uploaded to the temporary folder, they need to be moved to the permanent location and their paths stored in the database.
+
+You can manually implement the saving process yourself using model events, mutators or any other flavor that suits you, or use the `AjaxUploader` that we developed to help you with this task. 
+
+To enable the `AjaxUploader` just add `withFiles => true` to your field definition:
+
+```php
+[   
+    'name'  => 'photos',
+    'label' => 'Photos',
+    'type'  => 'dropzone',
+    'withFiles' => true
+],
+```
+
+To know more about the `withFiles`, how it works and how to configure it, [ click here to read the documentation ](https://backpackforlaravel.com/docs/6.x/crud-uploaders).
```

Comment:
Let's tell about the simple option first, then tell people about the complex use case:

```suggestion
At this point you have the dropzone field showing up, and the ajax routes setup to upload/delete files, but the process is not complete. Your files are now only uploaded to the temporary folder, they need to be moved to the permanent location and their paths stored in the database. The easiest way to fix that is to add `withFiles => true` to your field definition, this will use the standard `AjaxUploader` that Backpack provides:

```php
[   
    'name'  => 'photos',
    'label' => 'Photos',
    'type'  => 'dropzone',
    'withFiles' => true
],
```

Alternatively, you can manually implement the saving process yourself using model events, mutators or any other solution that suits you. To know more about the `withFiles`, how it works and how to configure it, [read its documentation](https://backpackforlaravel.com/docs/6.x/crud-uploaders).
```